### ISSUE TITLE
When I try to install Stegano, the following error shows up

### ISSUE THREAD

**Issue by simplyjustganesh7** [python-pillow/Pillow#5558]
### What are your OS, Python and Pillow versions?

* OS: Windows 10
* Python: 3.9.4
* Pillow: 8.2.0

```python
code goes here
```

**Comment by radarhere** [comment]
Hi. You haven't actually provided the error. I'm also not sure what a Stegano file is, unless you mean the Stegano library on PyPi?

**Comment by radarhere** [comment]
Ah, ok, I've just found your other post at https://github.com/jonathan-alvaro/python-stegano/issues/1.

So yes, this is about the stegano Python library.

Stegano requires Pillow 6.2.1 or 6.2.2, rather than the latest Pillow. However, you're trying to install this on Python 3.9, which Pillow didn't support until Pillow 8.0. So because there isn't a prebuilt wheel, pip is trying to build Pillow from source, and you don't have one of our dependencies, zlib, installed.

Possible solutions, from easiest to hardest -

Option 1) Switch to Python 3.8
- This should work for you straightaway. We provided Python 3.8 prebuilt wheels for Pillow 6.2.2.

Option 2) Ignore stegano's rule that it won't work with the latest Pillow, and see how you go
- Download stegano-0.9.8.tar.gz from https://pypi.org/project/stegano/#files
- Go into the directory. Open up setup.py and change "pillow>=6.2.1,<7.0.0" to just "pillow". Then run 'pip install .'

Option 3) Install zlib
- See https://zlib.net/

Option 4) Convince the maintainer(s) of Stegano to upgrade to a newer version of Pillow
- I'm not familiar with the stegano library, so I can't say how easy or hard that is, but this would be the most helpful option for other users like yourself in the future.
